tests/e2e/e2e_k8s_version_test.go

```diff
@@ -18,11 +18,11 @@ var _ = OSMDescribe("Test HTTP traffic for different k8s versions",
 		Bucket: 6,
 	},
 	func() {
-		Context("Version v1.22.0", func() {
-			testK8sVersion("v1.22.0")
+		Context("Version v1.22.8", func() {
+			testK8sVersion("v1.22.8")
 		})
-		Context("Version v1.19.7", func() {
-			testK8sVersion("v1.19.7")
+		Context("Version v1.21.11", func() {
+			testK8sVersion("v1.21.11")
```

Comment:
IIRC the intent here was for this to be the oldest version we support. Then this should be the latest v1.20 release.

Reply:
@nojnhuh there's no value testing v1.20.x. It has been dropped as well. The only reason I didn't deprecate support for v1.20.x from the Helm chart enforcement is that we avoid dropping installation support for multiple k8s versions in the same release. From a product perspective, there is absolutely no reason to test v1.20 in this test given we do not have product changes that use specific APIs from that version. This test was added as a best-effort validation to validate at least 2 prior versions.